doc/manual/installation/prerequisites-source.xml

```diff
@@ -9,6 +9,9 @@
 <itemizedlist>
 
   <listitem><para>GNU Make.</para></listitem>
+  
+  <listitem><para>Bash Shell. Bash, and no other shell, is required to
```

Comment:
"and no other shell" reads kind of oddly to me, can you  say something like "the configure script relies on bashisms" or some such?